modules/apps/document-library/document-library-preview-document/src/main/resources/META-INF/resources/preview/js/DocumentPreviewer.es.js

```diff
@@ -95,11 +95,19 @@ const DocumentPreviewer = ({baseImageURL, initialPage, totalPages}) => {
 		}, 100);
 	}
 
+	const createImageURL = (page) => {
+		const imageURL = new URL(baseImageURL);
+
+		imageURL.searchParams.set('previewFileIndex', page);
+
+		return imageURL.toString();
+	};
+
 	const loadPage = (page) => {
 		let pagePromise = loadedPages[page] && loadedPages[page].pagePromise;
 
 		if (!pagePromise) {
-			pagePromise = imagePromise(`${baseImageURL}${page}`).then(() => {
+			pagePromise = imagePromise(`${createImageURL(page)}`).then(() => {
```

Comment:
Please remove and use directly the function `imagePromise(createImageURL(page))`